packages/mds-db/geographies.ts

```diff
@@ -10,35 +10,53 @@ import { getReadOnlyClient, getWriteableClient } from './client'
 import { ReadGeographiesParams, PublishGeographiesParams } from './types'
 
 export async function readSingleGeography(geography_id: UUID): Promise<Geography> {
-  try {
-    const client = await getReadOnlyClient()
+  const client = await getReadOnlyClient()
 
-    const sql = `select * from ${schema.TABLE.geographies} where geography_id = '${geography_id}'`
-    const { rows } = await client.query(sql)
+  const sql = `select * from ${schema.TABLE.geographies} where geography_id = '${geography_id}'`
+  const { rows } = await client.query(sql)
 
-    const { id, ...geography } = rows[0]
-    return geography
-  } catch (err) {
-    await log.error('readSingleGeography', err)
-    throw new NotFoundError(`could not find geography ${geography_id}`)
+  if (rows.length === 0) {
+    await log.info(`readSingleGeography failed for ${geography_id}`)
+    throw new NotFoundError(`geography of id ${geography_id} not found`)
   }
+
+  const { id, ...geography } = rows[0]
+  return geography
 }
 
 export async function readGeographies(params: Partial<ReadGeographiesParams> = {}): Promise<Geography[]> {
   try {
     const client = await getReadOnlyClient()
 
-    const { get_read_only } = { get_read_only: false, ...params }
+    const { get_published, get_unpublished, geography_ids } = {
+      get_published: false,
+      get_unpublished: false,
+      ...params
+    }
+    if (get_published && get_unpublished) {
+      throw new BadParamsError('cannot have get_unpublished and get_published both be true')
+    }
 
     let sql = `SELECT * FROM ${schema.TABLE.geographies}`
 
     const conditions = []
     const vals = new SqlVals()
 
-    if (get_read_only) {
+    if (get_published) {
       conditions.push(`publish_date IS NOT NULL`)
     }
 
+    if (get_unpublished) {
+      conditions.push(`publish_date IS NULL`)
+    }
+
+    if (geography_ids) {
+      const SQLified_geography_ids = geography_ids.map(id => {
```

Comment:
@evanxadkins Does this trigger you?

Reply:
We have this in another part of the codebase
```
(${Object.values(
        event
      )
        .map(val => `'${val}'`)
        .join(',')})
```
It would be nice to have some kind of `mapArrayToSql(arr)` util to handle the `()` and stringification/join logic.